pages/common/speedtest.md

```diff
@@ -0,0 +1,33 @@
+# speedtest
+
+> Official command line interface for testing internet bandwidth using https://speedtest.net.
+> Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, see `speedtest-cli`.
```

Comment:
```suggestion
> Note: some platforms link `speedtest` to `speedtest-cli`, which is not the official CLI. If some of the examples in this page don't work, see `speedtest-cli`.
```

Reply:
I think is redundant with the first line of the command description.

Reply:
If you guys think so, that's fine.  A bit of redunancy isn't always a problem, but the section is a little full as it is, so maybe we can skip it.